Cargo.toml

```diff
@@ -232,6 +232,7 @@ pretty_assertions = { workspace = true }
 rstest = { workspace = true, default-features = false }
 serial_test = "3.1"
 tempfile = { workspace = true }
+iai-callgrind = "0.10.2"
```

Comment:
With the limits on only having `dev-dependencies`, how will this impact the test build times?

Reply:
comparing the timings from `cargo test --no-run --timings`
`main: 28.5s`
`this-PR: 29.5s`